design-library/src/components/BccGraphic/BccGraphic.stories.ts

```diff
@@ -0,0 +1,34 @@
+import BccGraphic from "./BccGraphic.vue";
+
+import type { Meta, StoryFn } from "@storybook/vue3";
+
+export default {
+  title: "Components/BccGraphic",
+  component: BccGraphic,
+  argTypes: {},
+} as Meta<typeof BccGraphic>;
+
+const Template: StoryFn<typeof BccGraphic> = (args) => ({
+  components: { BccGraphic },
+  setup() {
+    return { args };
+  },
+  template: `
+    <BccGraphic v-bind="args" />
+  `,
+});
+
+export const Example = Template.bind({});
+Example.args = {
+  bannerSrc: "https://event.bcc.no/wp-content/uploads/2023/10/Background-7.png",
+  logoSrc: "https://event.bcc.no/wp-content/uploads/2023/11/BUK4_Logo-Main.svg",
+  linkOut: true,
+  checked: true,
+  rounding: "xl",
+};
+
+export const ExampleWithCheck = Template.bind({});
+ExampleWithCheck.args = {
+  rounding: "md",
+  checked: true,
+};
```

Comment:
Rendering the story this way doesn't lead to nice code examples in Storybook:

![image](https://github.com/bcc-code/bcc-design/assets/39056299/019d3990-ed5f-45a2-a059-2aed55524bb0)

Instead for most stories, that will only have one or two props passed, I usually just render a completely custom story. See for example the Primary button story:

```ts
/**
 * Set the `variant` prop to `primary` to render a primary button. This is the default value for this prop so it's not necessary to set it.
 */
export const Primary: StoryFn<typeof BccButton> = () => ({
  components: { BccButton },
  template: `
    <div class="flex items-start gap-x-2">
      <BccButton size="xs">Primary (xs)</BccButton>
      <BccButton size="sm">Primary (sm)</BccButton>
      <BccButton size="base">Primary (base)</BccButton>
      <BccButton size="lg">Primary (lg)</BccButton>
      <BccButton size="xl">Primary (xl)</BccButton>
    </div>
  `,
});
```

I won't repeat it but this comment holds for all your stories 😅.

For the `Example` story, I usually add a custom code example, so that it displays a little nicer without the `template` tags or `v-bind`. It's a little more work but after the component is created those stories will probably not change a lot anyways.

All this is documented by the way, on one handy reference page: https://developer.bcc.no/bcc-design/design-library/writing-storybook-stories.html